src/Orleans/Configuration/ConfigUtilities.cs

```diff
@@ -614,12 +614,8 @@ public static string RuntimeVersionInfo()
         {
             var sb = new StringBuilder();
             sb.Append("   Orleans version: ").AppendLine(RuntimeVersion.Current);
-#if BUILD_FLAVOR_LEGACY
-            // TODO: could use Microsoft.Extensions.PlatformAbstractions package to get this info
             sb.Append("   .NET version: ").AppendLine(Environment.Version.ToString());
             sb.Append("   OS version: ").AppendLine(Environment.OSVersion.ToString());
-            sb.Append("   App config file: ").AppendLine(AppDomain.CurrentDomain.SetupInformation.ConfigurationFile);
```

Comment:
I'd leave the conditional compilation (although just around this line) if we didn't solve this yet, as a reminder of stuff we need to do before release (either decide to kill this or have some other way of troubleshooting, since it's not always straightforward to know which config file Orleans chose in some weird deployment cases)

Reply:
If the host runs full .net it has access to this class and property and can be logged when troubleshooting.
Since we're compiling against .Net Standard 2.0 we do not so I don't see the value of keeping the #if here, but of course if you insist I can add that back, please advise.